src/covariances.jl

```diff
@@ -0,0 +1,87 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENCE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    design_matrix(comps)
```

Comment:
Please rename to `designmatrix` without underscore. We only underscore when strictly needed to make it easy to read.

Reply:
Fixed in commit 88a467ad9064d35fa50fdc68ac48151cf66043a9.